### PR TITLE
samples: dect: add simple sample

### PR DESCRIPTION
**Pull request by eivindj-nordic**
Based on #15199

### CODE REVIEW

File: samples/dect/dect_phy/hello_dect/CMakeLists.txt
[the remark this answers is not in the document]
Comment (eivindj-nordic) [reply samples/dect/dect_phy/hello_dect/CMakeLists.txt:1]: I think it makes sense to keep it as is, we can discuss offline.

File: samples/dect/dect_phy/hello_dect/Kconfig [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config CARRIER
```
[the remark this answers is not in the document]
Comment (eivindj-nordic) [reply samples/dect/dect_phy/hello_dect/Kconfig:7]: I don't think that is needed. If so we should do that for other samples as well.

**Review by nordicjm — APPROVED** [review]
Sample files OK for using sysbuild

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -462,6 +462,11 @@ Debug samples
 
 |no_changes_yet_note|
 
+DECT samples
+------------
+
+* Added the :ref:`nrf_modem_dect_phy_hello` sample.
```
[the remark this answers is not in the document]
Comment (eivindj-nordic) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:468]: We plan to add more DECT samples in the future, so I think we could have a separate category.

**Review by divipillai — APPROVED** [review]

**Review by lemrey — APPROVED** [review]

**Review by lauri-piikivi — APPROVED** [review]
clean clear sample!

File: samples/dect/dect_phy/hello_dect/overlay-us.conf
[the remark this answers is not in the document]
Comment (eivindj-nordic) [reply samples/dect/dect_phy/hello_dect/overlay-us.conf:1]: updated, thanks.

**Comment by lemrey** [comment]
@eivindj-nordic can this be merged now?

**Comment by eivindj-nordic** [comment]
> @eivindj-nordic can this be merged now?

yes

**Pull request merged**